docs/libertyDev.md

```diff
@@ -97,15 +97,13 @@ test {
 
 ## libertyDevc Task, Container Mode
 
-The following is a technology preview. The features and parameters described below may change in future milestones or releases of the Liberty Gradle plugin.
-
 Start a Liberty server in a local container using the Dockerfile that you provide. An alternative to the `libertyDevc` task is to specify the `libertyDev` task with the `--container` option. 
 
-You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Gradle task. You can examine the commands used to build and run the container by viewing the console output of dev mode.
+You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Gradle task. The installed Docker Engine version must be 18.03.0 or higher.
 
-When dev mode runs with container support it still provides the same features. It monitors files for changes and runs tests either automatically or on demand. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. build/wlp/usr/servers/defaultServer/logs.
+When dev mode runs with container support, it builds a container image and runs the container. You can examine the commands that it uses to build and run the container by viewing the console output of dev mode. Additionally, it still provides the same features as the `libertyDev` task. It monitors files for changes and runs tests either automatically or on demand. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. build/wlp/usr/servers/defaultServer/logs.
```

Comment:
I remember we were saying the base image has to be the IBM Liberty image or one with the same layout (e.g. directory links like /config). If this is not stated elsewhere we should say it here. Also that it must be a Linux O/S image.

Reply:
Updated